code/modules/mob/living/blood.dm

```diff
@@ -85,6 +85,9 @@
 	if(!(NOBLOOD in dna.species.species_traits))
 		..()
 
+	else if(HAS_TRAIT(src, TRAIT_NOPULSE)) // Fulpstation Bloodsuckers edit: They don't need to be here
```

Comment:
so bloodsucker plasmamen are supposed to bleed blood?

Reply:
Plasmamen cant roll bloodsucker